python/cugraph/centrality/betweenness_centrality.py

```diff
@@ -197,7 +205,7 @@ def edge_betweenness_centrality(
 
     Returns
     -------
-    df : cudf.DataFrame
+    df : cudf.DataFrame or dictionary
```

Comment:
better if it is consistent with betweenness_centrality doc "df : cudf.DataFrame or Dictionary if using NetworkX"

Reply:
updated